server/server.js

```diff
@@ -1,20 +1,37 @@
 const express = require("express");
 const app = express();
-const session = require("express-session");
-app.use(
-  session({
-    secret: "keyboardkitteh",
-    resave: false,
-    saveUninitialized: true,
-    cookie: { maxAge: 60000 },
-  })
-);
+// const session = require("express-session");
+// app.use(
+//   session({
+//     secret: "keyboardkitteh",
+//     resave: false,
+//     saveUninitialized: true,
+//     cookie: { maxAge: 60000 },
+//   })
+// );
 
-//Database
-require("./server/config/mongoose.js");
+// Database
+// var mysql = require('mysql');
```

Comment:
Instead of var, you can use const instead. 

https://levelup.gitconnected.com/const-vs-let-vs-var-in-javascript-which-one-should-you-use-c56cf9b9e2a3